aggregator/aggregator.go

```diff
@@ -148,7 +148,7 @@ func (a *Aggregator) tryToSendVerifiedBatch(ctx context.Context, ticker *time.Ti
 
 	if proof != nil && proof.Proof != nil {
 		log.Infof("sending verified proof to the ethereum smart contract, batchNumber %d", batchNumberToVerify)
-		a.EthTxManager.VerifyBatch(batchNumberToVerify, proof.Proof)
+		a.EthTxManager.VerifyBatch(ctx, batchNumberToVerify, proof.Proof)
```

Comment:
Can we make this function return an error? The aggregator should know what to do if an inner error happened